prepare_data/src/main/groovy/org/orbisgis/osm/FormattingForAbstractModel.groovy

```diff
@@ -103,9 +103,9 @@ static IProcess transformRails() {
 
                     //special treatment if type is subway
                     if (type == "subway") {
-                        boolean keepSubway
-                        keepSubway = (row.getString("tunnel") != null && row.getString("tunnel") == "no" && row.getString("layer") != null && row.getString("layer").toInt() >= 0) || (row.getString("bridge") != null && (row.getString("bridge") == "yes" || row.getString("bridge") == "viaduct"))
-                        if (!keepSubway) {
+
+                        if (!((row.getString("tunnel") != null && row.getString("tunnel") == "no" && row.getString("layer") != null && row.getString("layer").toInt() >= 0)
+                                || (row.getString("bridge") != null && (row.getString("bridge") == "yes" || row.getString("bridge") == "viaduct")))) {
                             type = null
```

Comment:
Seems you can use here groovy sugar programming to reduce the code row.tunnel,  row.bridge...

Reply:
Done for all the processes